app/controllers/home_controller.rb

```diff
@@ -26,13 +26,7 @@ def show
   end
 
   def toggle_mobile
-    if session[:mobile_view].nil?
-      # we're most probably not on mobile, but user wants it anyway
-      session[:mobile_view] = true
-    else
-      # switch from mobile to normal html
-      session[:mobile_view] = !session[:mobile_view]
-    end
+    session[:mobile_view] = session[:mobile_view].nil? ? true : false
```

Comment:
I'm not sure this is equivalent.
```
new value:

old value | old variant | new variant
----------+-------------+------------
nil       | true        | true
true      | false       | false
false     | true        | false
```

If the tests pass, a test exercising the last case would be great though. Or maybe I'm wrong and it can never get `false`?